subsys/lorawan/Kconfig

```diff
@@ -44,35 +44,57 @@ config LORAWAN_PUBLIC_NETWORK
 	  Enable this option to use a public LoRaWAN network.
 	  Disable for private LoRaWAN networks.
 
-config LORAMAC_REGION_AS923
-	bool "Asia 923MHz Frequency band"
+choice
```

Comment:
The region configuration should be returned to how it was

Reply:
Done.